hack/ci-e2e.sh

```diff
@@ -58,38 +58,35 @@ minikube start
 minikube image load quay.io/metal3-io/baremetal-operator:e2e
 
 # Create libvirt domain
-VM_NAME="bmo-e2e-0"
-export BOOT_MAC_ADDRESS="00:60:2f:31:81:01"
+for vm_idx in $(seq 0 1); do
```

Comment:
```suggestion
for vm_idx in 0 1; do
```
Two item list does not warrant seq and a subshell.

Reply:
Thanks. My idea was to set any number of parallel tests, but still having a bit of dilemma.